libxcrypt.spec.rpkg

```diff
@@ -151,11 +151,6 @@ fi                                          \
 %global _ld_strict_symbol_defs 1
 
 
-# The library uses symbol versioning in way
-# that is not compatible with LTO currently.
-%undefine _lto_cflags
```

Comment:
Per @solardiz's concerns, let's _not_ do this yet.  Instead maybe just change the comment to "The library can be built with LTO using GCC >=10.2, but this mode is still experimental and not recommended for production."

Reply:
Well, the purpose of this file is to build a new rpm from each commit to `develop` *explicitly* for testing.  See: https://copr.fedorainfracloud.org/coprs/besser82/libxcrypt_CI/

Those builds need an explicit activation by the user and are otherwise NOT passed into production enviroments.

Reply:
Oh, OK.  I thought this was the spec file used for the builds actually going into Fedora.  I retract this comment.